.devcontainer/dev-container-setup.sh

```diff
@@ -6,20 +6,36 @@
 # SPDX-License-Identifier: EPL-2.0
 #
 
-GALASA_DEV_TOKEN_VALUE="$1"
-GALASA_API_SERVER_URL_VALUE="$2"
-SOURCE_MAVEN_VALUE="$3"
+function set_env_vars() {
+  local GALASA_DEV_TOKEN_VALUE="$1"
+  local GALASA_API_SERVER_URL_VALUE="$2"
+  local SOURCE_MAVEN_VALUE="$3"
+  local PROFILE_FILE="/home/node/.bashrc"
 
-PROFILE_FILE="/home/node/.bashrc"
+  if [ -n "$GALASA_DEV_TOKEN_VALUE" ]; then
+    echo "export GALASA_DEV_TOKEN=$GALASA_DEV_TOKEN_VALUE" >> "$PROFILE_FILE"
+  fi
 
-if [ -n "$GALASA_DEV_TOKEN_VALUE" ]; then
-  echo "export GALASA_DEV_TOKEN=$GALASA_DEV_TOKEN_VALUE" >> $PROFILE_FILE
-fi
+  if [ -n "$GALASA_API_SERVER_URL_VALUE" ]; then
+    echo "export GALASA_API_SERVER_URL=$GALASA_API_SERVER_URL_VALUE" >> "$PROFILE_FILE"
+  fi
 
-if [ -n "$GALASA_API_SERVER_URL_VALUE" ]; then
-  echo "export GALASA_API_SERVER_URL=$GALASA_API_SERVER_URL_VALUE" >> $PROFILE_FILE
-fi
+  if [ -n "$SOURCE_MAVEN_VALUE" ]; then
+    echo "export SOURCE_MAVEN=$SOURCE_MAVEN_VALUE" >> "$PROFILE_FILE"
+  fi
+}
 
-if [ -n "$SOURCE_MAVEN_VALUE" ]; then
-  echo "export SOURCE_MAVEN=$SOURCE_MAVEN_VALUE" >> $PROFILE_FILE
-fi
\ No newline at end of file
+function setup_python {
+  which pip3
+  rc=$?
+  if [[ "${rc}" != "0" ]]; then
+    error "pip3 tool is not available on your path. Install pip3 and re-try to install required python packages."
+    exit 1
+  fi
+  success "pip3 is available. OK"
```

Comment:
The `error` and `success` functions don't exist in this script, so they will return a `command not found` error